djautotask/api_rest.py

```diff
@@ -327,37 +327,49 @@ def get_headers(self):
 
         return headers
 
-    def _format_request_body(self, api_entity, changed_fields):
+    def _format_changed_fields(self, api_entity, changed_fields):
         body = {
             'id': api_entity.id
         }
 
         for field, value in changed_fields.items():
-
             if field in api_entity.EDITABLE_FIELDS:
                 key = api_entity.EDITABLE_FIELDS[field]
 
-                if isinstance(value, datetime.datetime):
-                    body.update({
-                        key: value.astimezone(
-                            pytz.timezone('UTC')).strftime(
-                                "%Y-%m-%dT%H:%M:%SZ")
-                    })
-
-                elif isinstance(value, models.Model):
-                    body.update({
-                        key: value.id
-                    })
-
-                elif isinstance(value, decimal.Decimal):
-                    body.update(
-                        {key: str(value) if value else '0'}
-                    )
+            body = self._format_request_body(body, key, value)
 
-                else:
-                    body.update(
-                        {key: str(value) if value else ''}
-                    )
+        return body
+
+    def _format_inserted_fields(self, inserted_fields):
```

Comment:
Editing and creating should be functionally the same operation right? Do we need this division of methods between updating or creating?